src/integration/pages/mediaAssetPage/user.js

```diff
@@ -0,0 +1,56 @@
+const { within } = require('@testing-library/dom');
+
+export default ({
+  headlineText,
+  timestamp,
+  bulletedListItem,
+  relatedContentHeadline,
+}) => {
+  [amp, canonical].forEach(page => {
+    describe(`And using ${page.platform}`, () => {
+      it('I can see the headline', () => {
+        const { getByText } = within(
+          page.document.querySelector('h1[id="content"]'),
+        );
+
+        const headline = getByText(headlineText);
+        expect(headline).toBeInTheDocument();
+      });
+
+      it('I can see the timestamp', () => {
+        const { getByText } = within(page.document.querySelector('time'));
+
+        const timestampEl = getByText(timestamp);
+
+        expect(timestampEl).toBeInTheDocument();
+      });
+
+      if (bulletedListItem) {
+        it('I can see the bulleted list item', () => {
+          const { getAllByRole } = within(page.document.querySelector('main'));
+          const bulletedLists = getAllByRole('list');
+          const [bulletedList] = bulletedLists.filter(list =>
+            list.innerHTML.includes(bulletedListItem),
+          );
+          const { getAllByText } = within(bulletedList);
+          expect(getAllByText(bulletedListItem)[0]).toBeInTheDocument();
+        });
+      } else {
+        it.skip('I can see the bulleted list item - no bulleted list exists on this page', () => {});
```

Comment:
I need a second opinion on this - not sure if we should do this or not.

Reply:
You mean the skip? Yeh I'm not sure about that. Nice idea but we're using a feature in a way that it's not intended to be used

Reply:
Removed